JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,15 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  let hours, minutes, second;
```

Comment:
i think we could try to mantain an standar, either use singular or plural

Reply:
since variables  hours, minutes or second  are not  being reassigned we could use const